src/arch/mpi-win-x86_64/charmrun

```diff
@@ -91,7 +89,7 @@ test $QUIET -eq 1 && charm_args=$charm_args" "++quiet
 
 args=$args" "$charm_args
 
-if [ "$DEBUG" = '1' -o "$DEBUG_NO_PAUSE" = '1' ]
+if [ "$DEBUG" = '1' ] || [ "$DEBUG_NO_PAUSE" = '1' ]
```

Comment:
What is the linting tool's rationale for this? It causes two process launches instead of one.

Reply:
The reason is here: https://github.com/koalaman/shellcheck/wiki/SC2166. This does indeed launch two processes instead of one, which is something we could fix with `[[` instead of `[`, but I think that might be better in another PR.